src/components/Fieldset/Fieldset.stories.tsx

```diff
@@ -45,12 +45,22 @@ export const FieldsetLegends: StoryObj<FieldsetLegendProps> = {
   ),
 };
 
-export const CheckboxField = CheckboxFieldStory;
-CheckboxField.args.fieldNote =
-  'Checkbox fieldset usage. See CheckboxField Stories for more examples.';
-CheckboxField.parameters = { chromatic: { disableSnapshot: true } }; // Disabled since already being snapped in original story.
+export const CheckboxField = {
```

Comment:
decided to convert this to CSF v3.0 while here (slightly easier to read) -- so also had to change the `CheckboxField` and `RadioField` stories to CSF v3